README.md

```diff
@@ -304,6 +304,13 @@ class ExampleSerializer(actorSystem: ExtendedActorSystem)
 Note that as with Serializability Checker and Dump Persistence Schema,
 this compiler plugin only runs in the sbt modules where `AkkaSerializationHelperPlugin` is explicitly enabled.
 
+To make this plugin work correctly, path to the source code directory where codecs are registered has to be set in sbt project settings - just add settings like below:
+```
+.settings(scalacOptions ++=
+    Seq(s"-P:codec-registration-checker-plugin:--source-code-directory=${baseDirectory.value.file.getAbsolutePath}/path/to/your/code/directory"))
```

Comment:
Huh that'll make the usage pretty complicated 🤔

Could you explain why passing the source code directory is needed to prevent the error?

Reply:
Also... even if it stays as is, I'd use a more realistic path as a sample (with `src/main/scala` etc.)

Reply:
> Huh that'll make the usage pretty complicated 🤔
> 
> Could you explain why passing the source code directory explicitly is needed to prevent the error?

Yes - because the only way to check if these missing types are "false-positives" or not - is by checking the source code in the repo that uses ASH plugins. This check does recursive search on all .scala files in the `--source-code-directory` for these possibly missing types. In order not to check whole repository (e.g. several modules in a bigger project) - it would be beneficial to specify the `--source-code-directory` - to check just the directory, where codecs are actually registered in code.

It might be also possible not to specify `path/to/your/code/directory` and to check whole repository by just checking ${baseDirectory.value.file.getAbsolutePath}
It would be one-step-easier for clients but in such situation - all .scala files from whole repo would be checked.

Unfortunately - this check has to be done on source code. Due to how incremental compilation works in sbt - we would not ba able to collect all "really existing" types using scala reflection.

Reply:
alternatively we can just set `${baseDirectory.value.file.getAbsolutePath}/src/main/scala` as a default ?

Reply:
(see the other comment about Hydra's case, for which this directory will **not** exist IIUC)

Reply:
It will work - see my reply on other comment - ${baseDirectory.value.file.getAbsolutePath} evaluates to the module (subproject) directory where plugin is enabled

Reply:
>  as a default ?

Okay... so will there be cases where ones will **not** want to simply set this flag to `src/main/scala` in the current subproject? 🤔

Reply:
Okay! pls proceed

Reply:
Check out `scalaSource.value`

Reply:
I just uploaded working solution - please see https://github.com/VirtusLab/akka-serialization-helper/pull/166/files#diff-d06aaa8abc0aa4f2b1333353fd3962771928ea9140a9ae91b0963b236bf2adceR58